### ISSUE TITLE
Documentation improvement suggestion: perhaps a few words about how uBlock differs, in particular in light of recent "anti-adblocking" code by people at facebook, twitch etc...

### ISSUE THREAD

**Issue by ghost** [uBlock-LLC/uBlock#1680]
Hi there,

This may be a bit lengthy but I try to make it relevant.

Today I decided to try out uBlock. The reason is quite simple - I was getting unwanted, hateful and useless ads by twitch.tv which has irritated me immensely. Not so much the fact that I have to pay 
for this irrelevant information that is rendered by my computer and paid by me but by the fact that
my adblocker did not block it.

This happens only rarely, so I was curious. After some googling around, I read that twitch uses
some other code checks to find out whether someone is using an adblocker (which is information 
that should be of no relevance to twitch.tv) but even more interestingly, to bypass adblocking and
attacking my computer with the malicious content.

As I was searching, on some websites people suggested various workarounds.

One was to not allow "acceptable ads" - actually I do not even know what the adblock team is trying to do here. There are no "acceptable" ads, this is propaganda; but more importantly, the workaround did not work anyway, I was still getting the malicious, hateful content by twitch.

Another website suggested uBlock as a better alternative and so I am giving it a try right now.

(I am using firefox + linux).

To be honest, though, I am getting a bit confused... what is the difference between adblock, adblock plus, ublock and ublock origin. But even more importantly, could someone actually add some documentation, even if it is just a small paragraph, about differences between these ad-blockers on different websites?

Some documentation how uBlock behaves on facebook or twitch would be helpful. (I do not use facebook myself but I do sometimes watch a few streamers on twitch - actually I think twitch needs to change the model entirely since while I have no problem to support individual streamers including donations, I have no interest in supporting twitch in any way whatsoever, not after they attacked me with the irrelevant ads that just waste my time.)

Anyway, TL;DR:

**- Please add some more documentation explaining how (and if) uBlock works for ad-blocking on twitch, facebook etc...; and whether there are any practical differences between uBlock and the other adblockers out there (since apparently adblock struggles with twitch, else I would not be here writing this comment/suggestion).**

Thanks for reading!
 uBlock as a

**Comment by ripdog** [comment]
Hello! This is the old, unmaintained uBlock. It does not have an acceptable ads program - that is a different addon, called 'Adblock Plus'.

This addon is simply broken. Switch to uBlock Origin, which is still actively and energetically maintained, and blocks ads on twitch!

https://github.com/gorhill/uBlock/releases

**Comment by anewuser** [comment]
@shevegen Please close this issue, as per above.
